saltstack/base/salt/prerequisites/usr/bin/ccmv2-helper.sh

```diff
@@ -33,12 +34,16 @@ setup_ccmv2() {
 
   INVERTING_PROXY_URL="$CCM_V2_INVERTING_PROXY_HOST"
 
+  CCM_V2_AGENT_ACCESS_KEY_ID=$(grep '^CCM_V2_AGENT_ACCESS_KEY_ID=' ${ACC_KEY_TMP_FILE} | cut -d'=' -f2)
+
   if [[ "$IS_CCM_V2_JUMPGATE_ENABLED" == "true" && "$IS_FREEIPA" == "true" && ! -z $CCM_V2_AGENT_ACCESS_KEY_ID ]]; then
-    ACCESS_KEY_ID="$CCM_V2_AGENT_ACCESS_KEY_ID"
+
+    CCM_V2_AGENT_ENCIPHERED_ACCESS_KEY=$(grep '^CCM_V2_AGENT_ENCIPHERED_ACCESS_KEY=' ${ACC_KEY_TMP_FILE} | cut -d'=' -f2)
```

Comment:
No need for grepping, the environment variables coming from the EC2 user data are not only visible in `user-data-helper.sh` but here as well.